api/can_api_v2_definition.py

```diff
@@ -61,6 +61,13 @@ class NonCovidPatientsMethod(enum.Enum):
 
 
 class DemographicDistributions(base_model.APIBaseModel):
+    """Distributions of demographic data.
+
+    Note that because different regions may have different demographic distributions for
+    the same field.  For instance, how age is collected is not uniform.
```

Comment:
'collected' makes me think of how ages are record from individuals but this is likely a decision made during some aggregation stage. How about "For instance, the age ranges are not uniform." ?